pkg/osbuild/monitor.go

```diff
@@ -123,22 +123,28 @@ func (sr *StatusScanner) Status() (*Status, error) {
 		Trace:   trace,
 		Message: msg,
 		Progress: &Progress{
-			Done:    status.Progress.Done,
-			Total:   status.Progress.Total,
-			Message: fmt.Sprintf("Pipeline %s", pipelineName),
+			Done:  status.Progress.Done,
+			Total: status.Progress.Total,
 		},
 		Timestamp: ts,
 		Duration:  time.Duration(status.Duration * float64(time.Second)),
 	}
 
+	if pipelineName != "" {
```

Comment:
A test case for this change wold be nice. In what situations does osbuild not give us a pipeline and/or a stage name? What does the progress struct look like in this case? Would be great to make sure that we provide still soemthing vaguely nice looking to the user and/or fix osbuild (if it needs fixing).

Reply:
Aha, nice. Thanks for this extra context!